index.html

```diff
@@ -1,11 +1,168 @@
+          </div>
+        </div>
+      </div>
+    </header>
+    <main class="main">
+      <div class="page__hero hero">
+        <h1 class="hero__title">Ефективні рішення для вашого бізнесу</h1>
+        <button class="button hero__button">Замовити послугу</button>
+      </div>
+
+      <section class="page__section features__section--pd-btm-0">
+        <div class="container">
+          <ul class="features__list">
+            <li class="features__item">
+              <h2 class="features__title">УВАГА ДО ДЕТАЛЕЙ</h2>
+              <p class="features__text">
+                Ідейні міркування, і навіть початок повсякденної роботи з
+                формування позиції.
+              </p>
+            </li>
+            <li class="features__item">
+              <h2 class="features__title">ПУНКТУАЛЬНІСТЬ</h2>
+              <p class="features__text">
+                Завдання організації, особливо рамки і місце навчання кадрів
+                тягне у себе.
+              </p>
+            </li>
+            <li class="features__item">
+              <h2 class="features__title">ПЛАНУВАННЯ</h2>
+              <p class="features__text">
+                Так само консультація з широким активом значною мірою зумовлює.
+              </p>
+            </li>
+            <li class="features__item">
+              <h2 class="features__title">СУЧАСНІ ТЕХНОЛОГІЇ</h2>
+              <p class="features__text">
+                Значимість цих проблем настільки очевидна, що реалізація
+                планових завдань.
+              </p>
+            </li>
+          </ul>
+        </div>
+      </section>
+
+      <section class="page__section services">
+        <div class="container">
+          <h3 class="services__title title">Чим ми займаємося</h3>
+          <ul class="services__list">
+            <li class="services__item">
+              <img
+                src="./images/Services/keyboard.jpg"
+                alt="keyboard"
+                class="services__image"
+              />
+            </li>
+            <li class="services__item">
+              <img
+                src="./images/Services/phone.jpg"
+                alt="phone"
+                class="services__image"
+              />
+            </li>
+            <li class="services__item">
+              <img
+                src="./images/Services/rpb.jpg"
+                alt="rpb"
+                class="services__image"
+              />
+            </li>
+          </ul>
+        </div>
+      </section>
+      <section class="page__section section__team--grey">
+        <div class="container">
+          <h4 class="team__title title">Наша команда</h4>
+          <ul class="team__list">
+            <li class="team__item">
+              <img
+                src="./images/Team/Ihor_Demyanenko.jpg"
+                alt=""
+                class="team__image"
+              />
+              <div class="team__border">
```

Comment:
this is not border. Would make more sense to name this class `team__item-content`